claimManagement/src/main/res/values/strings.xml

```diff
@@ -500,4 +500,47 @@
   <string name="empty_year">Please fill the year</string>
   <string name="DeleteFamilyOnlyOffline">Deleting of family will also Delete all Insurees, Policies and Premiums of this Family</string>
   <string name="LastInstallment">Last Installment</string>
+  <string name="title_activity_claim_review">ClaimReview</string>
```

Comment:
I added these keys to Lokalize. Please update the demo french language also.

Reply:
Added

Reply:
Are you sure you added all french translations? you can just dld the french strings file from Lokalize.